casbin/rbac/default_role_manager/role_manager.py

```diff
@@ -47,8 +47,9 @@ def clear(self):
 
     def add_link(self, name1, name2, *domain):
         if len(domain) == 1:
-            name1 = domain[0] + "::" + name1
-            name2 = domain[0] + "::" + name2
+            if len(domain[0]) > 1:
```

Comment:
When the length of domain name is 1, it won't work as expected.
But when I revert it., distributed enforcer broke.
@divy9881 Could you please work on this?